src/main.cpp

```diff
@@ -1,24 +1,25 @@
 #include "common.hpp"
+#include "core/commands/HotkeySystem.hpp"
 #include "core/filemgr/FileMgr.hpp"
+#include "core/frontend/Notifications.hpp"
 #include "core/hooking/Hooking.hpp"
 #include "core/memory/ModuleMgr.hpp"
 #include "core/renderer/Renderer.hpp"
-#include "game/frontend/GUI.hpp"
-#include "game/pointers/Pointers.hpp"
-#include "game/backend/ScriptMgr.hpp"
-#include "game/backend/FiberPool.hpp"
-#include "game/features/Features.hpp"
-#include "core/commands/HotkeySystem.hpp"
 #include "core/settings/Settings.hpp"
-#include "core/frontend/Notifications.hpp"
+#include "game/backend/FiberPool.hpp"
+#include "game/backend/ScriptMgr.hpp"
 #include "game/bigfeatures/CustomTeleport.hpp"
+#include "game/features/Features.hpp"
+#include "game/frontend/GUI.hpp"
+#include "game/pointers/Pointers.hpp"
+
 
 namespace YimMenu
 {
 	DWORD Main(void*)
 	{
-		const auto documents = std::filesystem::path(std::getenv("USERPROFILE")) / "Documents";
-		FileMgr::Init(documents / "HellBase"); // TODO
+		const auto documents = std::filesystem::path(std::getenv("appdata")) /= "HorseMenu";
```

Comment:
Heh, why is there a `/=` in the middle of it? Seems like UB, I'm specifically talking about `=` coming after the `/`.

Reply:
What does UB mean? i'm a little out of touch with some programmer slang. The /= does the same thing as just /